docs/blog/2020-06-08-blog-2.0/index.md

```diff
@@ -0,0 +1,116 @@
+
+## Changing the look is easier than ever before
+
+Being able to shadow styles has always been a great way to get granular with your Theme UI theme, but not everyone wants to do that! I'm not much of a designer myself and I wanted to change the look of my site without having to pick out every color.
+
+### Presets
+
+In Blog Theme 2.0 you can pass in a Theme UI preset to style your site. The default is `gatsby-theme-ui-preset` which looks and feels much like the existing blog theme styles, but there are a number of others open to you!
+
+You can make use of those [provided by Theme UI](https://theme-ui.com/packages/presets), or try out something from the larger community, like [sketchy-preset](https://github.com/beerose/theme-ui-sketchy).
+
+All of these presets act as base styles and your local shadowed styles are automatically deep merged with them, with local styles taking precedent.
+
+If you'd rather not use any preset and set everything locally yourself, you can set `preset` to `false`.
+
+### Prism
+
+Another option for presets are prism highlighting. Theme UI comes with a number of built-in [prism presets](https://theme-ui.com/packages/prism#syntax-themes) that you can reference directly by name and the theme will do the rest for you.
+
+```
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-theme-blog`,
+      options: {
+        prismPreset: `prism-okaidia`, // highlight-line
+      },
+    },
+  ],
+}
+```
+
+All prism presets take precedence over styles in your `preset` theme, but are less specific than local shadowing.
+
+## The new `gatsby-theme-blog-core`
+
+In addition to all the changes above, we've also made some adjustments to the architecture in `gatsby-theme-blog-core`. It now comes with built-in components for displaying an individual blog post or a blog post listing. These components are completely unstyled but elements within them have classNames so you can change their look without any shadowing at all.
+
+This gives you more tools than ever before to use this theme as an add-on for site functionality.
+
+## `gatsby-theme-blog-darkmode`
+
+Another change is the darkmode toggle that's long been a staple of the blog theme. In order to make styling a bit more fun and remove the need for a darkmode out of the box, this is now a separate theme!
+
+Use it in combination with `gatsby-theme-blog` and take advantage of the same functionality you've come to expect!
+
+## Next Steps
+
+We're so excited to see you make use of these updated themes and can't wait to see what the community builds. As always, we'd love for you to share it with us on twitter!
```

Comment:
Twitter capitalized so we don't hear from our dear German friend. Also probably link or @GatsbyJS mention?